lib/template-dependencies.js

```diff
@@ -82,6 +70,29 @@ ${formattedError}`,
     .catch(handleInternetAccessError);
 }
 
+function formatElmJsonError(error, options) {
+  if (error.stderr === undefined) {
+    return error.message;
+  }
```

Comment:
You never know what you catch in JS. Just in case this error didn’t come from our handler which attaches `.stderr`, return the error message rather than crashing (“cannot call .toString() on undefined”.)